### PR TITLE
More localization!

### PR DESCRIPTION
**Pull request by Tanza3D**

### CODE REVIEW

File: global/lang/en_GB/navbar.json [hunk 1]
```diff
@@ -26,14 +30,22 @@
...
     "tooltip.settings": "Settings",
-    "tooltip.search": "Search"
+    "tooltip.search": "Search",
+    "otherApps.title": "other pages",
+    "otherApps.back": "back to apps",
+    "otherApps.translators": "Translators",
+    "otherApps.userGroups": "User Groups",
+    "misc.cantContactOsu": "Osekai cannot contact osu!'s servers. This may be because osu! is down, or you could be connected to a private server. If you are connected to a private server, please disconnect from it.",
+    "misc.restriction.title": "Your account on Osekai has been restricted.",
+    "misc.restriction.description": "To appeal this, please contact us on the <a href=\"https://discord.com/invite/8qpNTs6\">osu! Medal Hunters Discord server</a>"
```
Comment (minusQuantumNeko) [inline global/lang/en_GB/navbar.json:50]:
```suggestion
    "misc.restriction.description": "To appeal this, please contact us on the <a href=\"https://discord.com/invite/8qpNTs6\">osu! Medal Hunters Discord</a> server"
```

**Review by minusQuantumNeko — CHANGES_REQUESTED** [review]

File: global/lang/en_GB/navbar.json (hunk 1 above)
Comment (Tanza3D) [reply global/lang/en_GB/navbar.json:50]: this doesn't make sense?

File: global/css/main.css [hunk 2]
```diff
@@ -724,7 +724,7 @@ a {
 .osekai__panel-nav__lrcont_lmc {
     padding: 0px;
     display: flex;
-}
+} /* hehe */
```
Comment (EXtremeExploit) [inline global/css/main.css:727]: WYSI OMG THE FUNNY NUMBER WHEN YOU SEE IT 727 OMG

File: global/css/main.css [hunk 3]
```diff
@@ -3072,7 +3072,7 @@ option {
...
+    -webkit-clip-path: polygon(40% 7.67949%, 43.1596% 6.20615%, 46.52704% 5.30384%, 50% 5%, 53.47276% 5.30384%, 56.8404% 6.20615%, 60% 7.67949%, 81.65064% 20.17949%, 84.50639% 22.17911%, 86.97152% 24.64425%, 88.97114% 27.5%, 90.44449% 30.6596%, 91.34679% 34.02704%, 91.65064% 37.5%, 91.65064% 62.5%, 91.34679% 65.97296%, 90.44449% 69.3404%, 88.97114% 72.5%, 86.97152% 75.35575%, 84.50639% 77.82089%, 81.65064% 79.82051%, 60% 92.32051%, 56.8404% 93.79385%, 53.47296% 94.69616%, 50% 95%, 46.52704% 94.69616%, 43.1596% 93.79385%, 40% 92.32051%, 18.34936% 79.82051%, 15.49361% 77.82089%, 13.02848% 75.35575%, 11.02886% 72.5%, 9.55551% 69.3404%, 8.65321% 65.97296%, 8.34936% 62.5%, 8.34936% 37.5%, 8.65321% 34.02704%, 9.55551% 30.6596%, 11.02886% 27.5%, 13.02848% 24.64425%, 15.49361% 22.17911%, 18.34936% 20.17949%);
```
Comment (EXtremeExploit) [inline global/css/main.css:3075]: MY GOD OMG

**Review by EXtremeExploit — APPROVED** [review]
k

**Pull request merged**